### DISCUSSION TITLE
Moving to Appolo from Sunshine (AIO)

### DISCUSSION THREAD

**Discussion by BehelitSkull** [ClassicOldSong/Apollo#153] [Q&A]
Hello !

Fist thanks for your work on Apollo i already find it better than Classic Sunshine.

To explain my situation, i was using Sunshine AIO, a tool that installs many programs : Playnite, Virtual Display Driver that is disabled in device manager by default, with scripts/commands that activate it when i start streaming (and that disable the laptop screen) and disable it when i stop streaming. 

There's also a vsync toggle program that stop vsync when the stream starts, nefcon, Playnite Watcher, MultiMonitorTool, Sunshine Virtual Monitor.

Now when i installed Apollo i didn't uninstall anything before except the Virtual Display Driver as recommended.

In the Apps list of Apollo i lost all apps and games except Steam and Desktop. I tried to use Sunshine App Export in Playnite but it didn't work. I don't know if Apollo use the same "apps.json" that Sunshine uses ? I found an apps.json in AppData/local/temp. 

I also lost the commands in Server Commands. How can i get back all my apps ? And which program can i discard that is no longer necessary for Apollo ?

**Comment by ClassicOldSong** [comment]
You can copy the `apps.json` from Sunshine installation to Apollo's, Apollo will convert them automatically. To do that, simply copy the `apps.json` from your Sunshine's installation into Apollo's, and restart Apollo.

Server Commands are Apollo's new feature that requires Artemis to function. You'll find that there's no such fields in Sunshine's config.
